spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -1047,6 +1048,8 @@ public void setRequestAttributesEnabled(boolean requestAttributesEnabled) {
 		 */
 		private Integer selectors;
 
+		private Accesslog accesslog;
```

Comment:
If so we cannot have anymore a setAccesslog method. Why it should be final?
I changed that but I'd really like to understand why?